# apple clang 13 has a bug where [[likely]] or [[unlikely]] returns
# don't count for checking that a constexpr function has a return,
# so at least one return must be unmarked
--- a/src/robusttransaction.cxx
+++ b/src/robusttransaction.cxx
@@ -62,9 +62,8 @@ constexpr tx_stat parse_status(std::string_view text) noexcept
   case 'i':
     if (text == in_progress)
       PQXX_LIKELY return tx_in_progress;
     break;
   }
-  PQXX_UNLIKELY
   return tx_unknown;
 }
 
